api/src/test/java/ca/bc/gov/educ/api/distribution/service/DistributionServiceTest.java

```diff
@@ -90,59 +90,59 @@ public class DistributionServiceTest {
 	public void testdistributeCredentialsMonthlyLocalDownload() {
-		DistributionResponse res = testdistributeCredentials_transcript("MER","USERDIST",false,"Y");
+		DistributionResponse res = testdistributeCredentials_transcript("MER","USERDIST",false,"Y", false);
 		assertNotNull(res);
-		res = testdistributeCredentials_certificate("MER","USERDIST","YED2",null,false);
+		res = testdistributeCredentials_certificate("MER","USERDIST","YED2",null,false, false);
 		assertNotNull(res);
-		res = testdistributeCredentials_certificate("MER","USERDIST","YEDB",null,false);
+		res = testdistributeCredentials_certificate("MER","USERDIST","YEDB",null,false, false);
 		assertNotNull(res);
-		res = testdistributeCredentials_certificate("MER","USERDIST","YEDR",null,false);
+		res = testdistributeCredentials_certificate("MER","USERDIST","YEDR",null,false, false);
 		assertNotNull(res);
 	}
 
 	@Test
 	public void testdistributeCredentialsMonthly_schoolNull() {
-		DistributionResponse res = testdistributeCredentials_transcript("MER","USERDIST",true,null);
+		DistributionResponse res = testdistributeCredentials_transcript("MER","USERDIST",true,null, false);
 		assertNotNull(res);
-		res = testdistributeCredentials_certificate("MER","USERDIST","YED2",null,false);
+		res = testdistributeCredentials_certificate("MER","USERDIST","YED2",null,false, false);
 		assertNotNull(res);
-		res = testdistributeCredentials_certificate("MER","USERDIST","YEDB",null,false);
+		res = testdistributeCredentials_certificate("MER","USERDIST","YEDB",null,false, false);
 		assertNotNull(res);
-		res = testdistributeCredentials_certificate("MER","USERDIST","YEDR",null,false);
+		res = testdistributeCredentials_certificate("MER","USERDIST","YEDR",null,false, false);
 		assertNotNull(res);
 	}
 
 	@Test
 	public void testdistributeCredentialsUserReq() {
-		DistributionResponse res = testdistributeCredentials_certificate("MER","USERDISTRC","YED2","John Doe",false);
+		DistributionResponse res = testdistributeCredentials_certificate("MER","USERDISTRC","YED2","John Doe",false, false);
 		assertNotNull(res);
-		res = testdistributeCredentials_certificate("MER","USERDISTRC","YEDB","John Doe",false);
+		res = testdistributeCredentials_certificate("MER","USERDISTRC","YEDB","John Doe",false, false);
 		assertNotNull(res);
-		res = testdistributeCredentials_certificate("MER","USERDISTRC","YEDR","John Doe",false);
+		res = testdistributeCredentials_certificate("MER","USERDISTRC","YEDR","John Doe",false, false);
 		assertNotNull(res);
 	}
 
 	@Test
 	public void testdistributeCredentialsYearly() {
-		DistributionResponse res = testdistributeCredentials_transcript("MERYER","USERDIST",false,null);
+		DistributionResponse res = testdistributeCredentials_transcript("MERYER","DISTRUN_YE",false,null, true);
 		assertNotNull(res);
-		res = testdistributeCredentials_certificate("MERYER","USERDIST","YED2",null,false);
+		res = testdistributeCredentials_certificate("MERYER","DISTRUN_YE","YED2",null,false, true);
```

Comment:
Getting unit test failure here (line 141)

Reply:
The activity code, DISTRUN_YE is right for Year-end dist run.   The wrong one was used.